cmd/runtime_options_test.go

```diff
@@ -233,6 +234,7 @@ func testRuntimeOptionsCase(t *testing.T, tc runtimeOptionsTestCase) {
 	fs := afero.NewMemMapFs()
 	require.NoError(t, afero.WriteFile(fs, "/script.js", jsCode.Bytes(), 0644))
 	runner, err := newRunner(
+		logrus.StandardLogger(),
```

Comment:
Instead of `logrus.StandardLogger()`, can you use [this](https://github.com/loadimpact/k6/blob/master/lib/testutils/test_output.go#L28-L41) in the tests, with a small `newLogger` helper that uses it as output? Or even better, make a new implementation of [`logrus.FieldLogger`](https://godoc.org/github.com/sirupsen/logrus#Logger) that uses [`testing.TB`](https://golang.org/pkg/testing/#TB)?

This would hopefully greatly reduce the logging noise whenever a test fails

Reply:
I prefer this to be done in a separate PR ... as this is likely to take a lot of time and half of those changes are in multiple modules and more importantly not just in tests ... am literally blocked and have already made one of my PRs not mergeable (probably) because of those changes.

So I prefer if at later time someone (including me) fixes those across the board, again probably in not just one PR across the whole source base